builder/amazon/common/step_run_source_instance.go

```diff
@@ -116,6 +117,26 @@ func (s *StepRunSourceInstance) Run(ctx context.Context, state multistep.StateBa
 		EbsOptimized:        &s.EbsOptimized,
 	}
 
+	if s.NoEphemeral {
+		// This is only relevant for windows guests. Ephemeral drives by
```

Comment:
> This is only relevant for windows guests.

Should we also add a line about this in the docs ?

Reply:
Good point.